
robot_name: "main_robot"

first_puck_zone: [0.6, 1.34, 1.57]
second_puck_zone: [0.8, 1.34, 1.57]
third_puck_zone: [1, 1.34, 1.57]



forth_puck: [0.6, 1.34, 1.57]

fifth_puck: [0.6, 1.34, 1.57]


# Orange zone
# red_cell_puck: [2.5, 0.45]
# green_cell_puck: [2.5, 0.75]
# blue_cell_puck: [2.5, 1.05]

# Purple zone
red_cell_puck: [0.5, 0.45]
green_cell_puck: [0.5, 0.75]
blue_cell_puck: [0.5, 1.05]
